src/lua/scripts/inspect.lua

```diff
@@ -0,0 +1,335 @@
+local inspect ={
+  _VERSION = 'inspect.lua 3.1.0',
```

Comment:
Maybe you also want to upgrade the embedded script?

https://github.com/kikito/inspect.lua/tags

Reply:
done